### PR TITLE
Generate random string

### PR DESCRIPTION
**Pull request by nir1212**
## Status
Ready

## Description
Generate Random String

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

## Required version of Demisto
any

## Does it break backward compatibility?
   - No

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

**Comment by nir1212** [comment]
@anara123 1. i dont see "GenerateUUID" on my instance
2.i dont need uuid , i need password generator and the one we got is wired and not so useful 
cant say the length i need on it and i must give it min and max on every char type.

this one is far more simple

File: Scripts/script-GenerateRandomString.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+          if digits == "True":
+              characters += string.digits
+
+          if punctuation == "True":
+              characters += string.punctuation
+
+      else:
+          demisto.results(raise_error("Punctuation,Digits or Letters must be True."))
+          sys.exit(1)
```
Comment (yuvalbenshalom) [inline Scripts/script-GenerateRandomString.yml]: remove sys.exit
better use return

File: Scripts/script-GenerateRandomString.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  else:
+      demisto.results(raise_error("Length must be grather than 0."))
+      sys.exit(1)
```
Comment (yuvalbenshalom) [inline Scripts/script-GenerateRandomString.yml]: remove sys.exit
better use return

File: Scripts/script-GenerateRandomString.yml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          'ContentsFormat': formats['json'],
+          'Type': entryTypes['note'],
+          'Contents': data,
+          'ReadableContentsFormat': formats['markdown'],
+          'HumanReadable': tableToMarkdown(title, data) if data else 'No result were found',
+          'EntryContext': ec
+      }
+
+
+  def raise_error(error):
```
Comment (yuvalbenshalom) [inline Scripts/script-GenerateRandomString.yml]: change to create_error_entry

**Pull request merged**